src/Microsoft.Health.Fhir.Core/Messages/Export/CreateExportRequest.cs

```diff
@@ -6,23 +6,27 @@
 using System;
 using EnsureThat;
 using MediatR;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export;
 using Microsoft.Health.Fhir.Core.Models;
 
 namespace Microsoft.Health.Fhir.Core.Messages.Export
 {
     public class CreateExportRequest : IRequest<CreateExportResponse>
     {
-        public CreateExportRequest(Uri requestUri, string resourceType = null, PartialDateTime since = null)
+        public CreateExportRequest(Uri requestUri, ExportJobType requestType, string resourceType = null, PartialDateTime since = null)
         {
             EnsureArg.IsNotNull(requestUri, nameof(requestUri));
 
             RequestUri = requestUri;
+            RequestType = requestType;
```

Comment:
The reason we initially added `resourceType` here was to have support for `Patient` and other resources in the future. Since `ExportJobType` is being used to track that, it seems like we should just remove `resourceType` from here and downstream (`ExportJobRecord`). I don't think it's being used anywhere downstream either with the current change. Will make things cleaner since we can avoid confusion on whether we have to use `ExportJobType` or `resourceType`

Reply:
When support for the _type parameter is added resource type will still be needed to pass that information. It isn't being used right now, but _type is the next story to be worked on so I don't want to remove it here.